test/datasets/datasets_test.py

```diff
@@ -35,26 +35,3 @@ def test_speechcommands(self):
     def test_cmuarctic(self):
         data = CMUARCTIC(self.path)
         data[0]
-
-
-class TestCommonVoice(TorchaudioTestCase):
-    backend = 'default'
-    path = get_asset_path()
-
-    def test_commonvoice(self):
-        data = COMMONVOICE(self.path, url="tatar")
-        data[0]
-
-    def test_commonvoice_diskcache(self):
```

Comment:
Can you migrate these two tests that use `disckcache_iterator` and `bg_iterator` to `test/datasets/utils_test.py`?

`class TestIterator(TorchaudioTestCase)` then add as methods `test_disckcache_iterator` and `test_bg_iterator`. The contents of these tests can stay same.